tasks/CleanTable.tex

```diff
@@ -1,80 +1,72 @@
+	
+	\item \textbf{Table Setting:} Placed on the table are: 
 	\begin{itemize}[nosep]
-		\item\textit{Silverware}: Any two objects (fork, knife, spoon).
-		\item\textit{Tableware}: Any three objects, excluding silverware. At least one must be a dish.
-		\item\textit{Dishwasher Tab}: One Dishwasher Tab.
+		\item 8 objects with 4 being cutlery and 4 tableware (see~\ref{sec:rules:scenario:objects}).
+		\item One dishwasher tab.
 	\end{itemize}
+	
+	\item \textbf{Dishwasher:} The team chooses whether the dishwasher is open and whether the racks are pulled out.
+	
+	\item \textbf{Safe Placing:} Objects must be placed with care. It must be clear that the robot is trying to place the object, not throwing or dropping it.
 \end{itemize}
 
 
-\subsection*{Additional rules and remarks}
+\subsection*{Additional Rules and Remarks}
 \begin{enumerate}[nosep]
-	\item \textbf{Deus ex Machina:} The following reductions apply:
+	\item \textbf{Placing Objects:} Since placing objects is the main focus of the test, asking a human to place something is not allowed.
+	
+	\item \textbf{Partial Scoring:} The main task allows partial (per object) scoring.
+	
+	\item \textbf{Deus ex Machina:} Score reduction applies per object as follows:
 	\begin{itemize}[nosep]
-		\item \textbf{Handover:} Handing over an object causes a score reduction of 100pts.
-		\item \textbf{Showing objects:} Pointing, or telling to the robot where an object is causes a score reduction of 50pts.
-		\item \textbf{Placing tableware:} Having a human assistant placing tableware inside the dishwasher causes a score reduction of 200pts.
-		\item \textbf{Placing cutlery:} Having a human assistant placing cutlery inside the dishwasher causes a score reduction of 250pts.
-	\end{itemize}
-
-	\item \textbf{Safe placing:} Objects must be placed with care. It must be clear that the robot is trying to place the object, not throwing or dropping it.
+		\item \textbf{Request Object Handover}: Asking to hand over an object causes a score reduction of 75pts.
+		\item \textbf{Request Object Locations by Gesture:} Pointing at an object to be picked up or where to place an object causes a reduction of 40pts.
 
-	\item \textbf{Dishwasher:} The team decides whether the robot will place the objects in the dishwasher's rack or in the official tray.
-
-	\item \textbf{Dishwasher door:} The dishwasher door is open and with the racks pulled out by default.
-	The team leader can, however, request the dishwasher to be closed and score additional points for opening it. If the robot fails to open the door, it must clearly state it and request the referee to open it.
+	\end{itemize}
 
-	% \item \textbf{Award:} Teams that carry out this task are eligible to win the \textit{Procter \& Gamble Dishwasher Challenge Award}, described in \refsec{award:pandg}.
+	\item \textbf{Award:} Teams that carry out this task are eligible to win the \PGAward{} and a prize sponsored by P\&G. The winner is chosen by representatives of P\&G.
```

Comment:
We don't know any criteria the P&G representative is using for choosing a winner. Shall we ask them to be more transparent or leave it as is like was in the OpenChallenge?

Reply:
You should ask the EC if they can ask for more detailed criteria. Otherwise, I'd guess that representatives will explain it at the competition if necessary.